assets/charts/components/headlamp/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: headlamp
+description: An easy-to-use and versatile dashboard for Kubernetes brought to you by Kinvolk.
+type: application
+
+version: 0.1.0
+appVersion: 0.0.0
```

Comment:
I wonder if this should at least be 0.0.1, even though this is our own chart

Reply:
I guess it's `0.0.0` because Headlamp currently do not have any public release (it should though!).

Reply:
It will soon :)